distributed/security.py

```diff
@@ -28,6 +28,11 @@ class Security:
     tls_ciphers : str, optional
         An OpenSSL cipher string of allowed ciphers. If not provided, the
         system defaults will be used.
+    tls_min_version : ssl.TLSVersion, optional
+        The minimum TLS version to support. Defaults to TLS 1.2.
+    tls_max_version : ssl.TLSVersion, optional
+        The maximum TLS version to support. Defaults to the maximum version
+        supported.
```

Comment:
Somewhat of a nit, but `tls_version_min` / `tls_version_max` would be more consistent with the convention used for other parameter names (e.g. `tls_client_cert` / `tls_client_key`)

Reply:
That's only true for things specific to client/worker/scheduler. This is more similar to `tls_ca_file`/`tls_ciphers`, which just forward kwargs on to the underlying ssl object. I think the kwarg names are fine as is.